Update scim.md (#8350)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim.md

- Group creation and management
- Importing groups and users

> When SCIM is enabled, the following functionality will change: 

Reviewer: ```suggestion
When SCIM is enabled, the following changes apply:
```

> - Users will not be added to default groups automatically

Reviewer: ```suggestion
- Users are not automatically added to default groups
```

- Manual actions such as inviting users, updating user information and updating group memberships will be disabled by default

> - SSO group mappings will be disabled in favor of SCIM group management

Reviewer: ```suggestion
- SSO group mappings are disabled in favor of SCIM group management
```
